msctl

```diff
@@ -2303,7 +2303,9 @@ MAPS_URL=$(getDefaultsValue 'mscs-maps-url' 'http://minecraft.server.com/maps')
 
 # Respond to the command line arguments.
 # ---------------------------------------------------------------------------
-case "$1" in
+# Stores the user-issued command.
+command=$1
```

Comment:
you could add a `shift` command here since you are storing `$1` in `$command`.  This would save you from having to use `shift` in each section below.

Reply:
Also, every other variable in the script is in all caps. To match the rest of the script, it should be:
```
COMMAND=$1
```
This is just esthetics

Reply:
thanks for the comments :) 

> you could add a `shift` command here since you are storing `$1` in `$command`. This would save you from having to use `shift` in each section below.

the reason i didn't do this is because there are other parts of the script where they only accept one argument, so I'd have to shift all of those arguments down by 1. for instance, right now to create a world it calls `createWorld $2 $3 $4`. if I had the shift command applied globally, then I'd have to modify that statement so it's `createWorld $1 $2 $3`, and do this with several other statements in the file. Do you think its better to shift the parameters globally and just make these tweaks, or keep it as it is right now (where it's only shifted in certain code blocks)? 

> Also, every other variable in the script is in all caps. To match the rest of the script, it should be:
> 
> ```
> COMMAND=$1
> ```
> This is just esthetics

okay, i wasn't sure if it was a style thing or if the caps actually denoted a constant or something

Reply:
I'm fine leaving this patch as is, although it does need to be updated to resolve the conflict created by not sorting the `case` statement options.

Reply:
i think i resolved all the issues with the latest commit =)